lib/checkstl.cpp

```diff
@@ -2966,4 +2966,3 @@ void CheckStl::checkMutexes()
         }
     }
 }
-
```

Comment:
please revert this change.

Reply:
this file has 2 empty lines at the end, 1 removed, 1 left intact.

Ok, I'll return an extra empty line